examples/snapshot/basic_snap.py

```diff
@@ -0,0 +1,47 @@
+""" Sample script to demonstrate use of SoCo snapshot
+
+This is useful for scenarios such as when you want to switch to radio,
+an announcement or doorbell sound and then back to what was playing previously.
+
+To use it just change the IP address to one of your Sonos players. 
+
+This script will:
+ - start playing a radio station
+ - take a snapshot of the status of the sonos player (what playing, volume etc.)
+ - play an alert mp3 for a few seconds (a poem!)
+ - re-instate the sonos player to it's previous state (the radio station)
+ 
+Note: the snap function is designed to be used once, Having taken a snap to 
+take another re-instantiate the class.
+"""
+
```

Comment:
A comment about py2 and 3 compatible code. I guess this was developed on Py3 as you use print as a function. However, the `print()` only work like it is supposed to in Py2, if there is only one argument. If doing `print("more", "strings")` it will print out `("more", "strings")`. The proper way to do it is to start the file with: `from __future__ import print_function`. Then print will actually be a function also in Py2.

While you are at it, you might as well also add `from __future__ import print_function, unicode_literals`. Then you will not need to preface your strings with u to make sure they are unicode.